src/Microsoft.TestPlatform.Build/Resources/Resources.resx

```diff
@@ -126,4 +126,7 @@
   <data name="TestRunningSummary" xml:space="preserve">
     <value>Test run for {0}({1})</value>
   </data>
+  <data name="UpdateTestSdkForCollectingCodeCoverage" xml:space="preserve">
```

Comment:
@pvlakshm Please review

Reply:
Please use "Update the Microsoft.NET.Test.Sdk package reference to version 15.8.0 or later to collect code coverage."

Reply:
Done.